astropy/nddata/mixins/ndarithmetic.py

```diff
@@ -23,212 +168,638 @@ class NDArithmeticMixin(object):
     When subclassing, be sure to list the superclasses in the correct order
```

Comment:
Is there a way to automate or force the correct order? @embray you know a lot of python magic.